src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -100,6 +102,8 @@ public void launch(SlaveComputer computer, TaskListener listener) {
         final PodTemplate template = slave.getTemplate();
         try {
             KubernetesClient client = slave.getKubernetesCloud().connect();
+
+            ArrayList<PersistentVolumeClaim> pvcs = template.buildPVC(client, slave);
```

Comment:
General comment: always prefer interfaces (`List`) to implementations (`ArrayList`)